scss/_patterns_side-navigation.scss

```diff
@@ -644,18 +588,6 @@
     color: $color-sidenav-text-active;
   }
 
-  .p-side-navigation__expand {
```

Comment:
These color theme styles seem to be missing now from the new extracted files.

It won't be trivial to move them though, I guess the only reliable way would be to create a minimal theme in the expandable SCSS file as well.

Reply:
Done

Reply:
I added a new docs page for the dark variant [/docs/examples/patterns/side-navigation/expandable-dark](https://vanilla-framework-4536.demos.haus/docs/examples/patterns/side-navigation/expandable-dark) to make it easier to test.